modules/nvidia_plugin/src/cuda_plugin.cpp

```diff
@@ -25,6 +25,7 @@
 #include "nvidia/nvidia_config.hpp"
 #include "openvino/runtime/properties.hpp"
 #include "cpp_interfaces/interface/ie_internal_plugin_config.hpp"
+#include "any_copy.hpp"
```

Comment:
it's located outside of developer API (part of src inference folder), so cannot be used here